# name of the joint
string joint
# the numerical id of the joint
uint16 joint_id
# position of the joint
# unit - radian
float64 position
# velocity of the joint
# unit - radian per second
float64 velocity
